vt/client.py

```diff
@@ -44,7 +44,13 @@
 
 def _make_sync(future):
   """Utility function that waits for an async call, making it sync."""
-  return asyncio.get_event_loop().run_until_complete(future)
+  try:
```

Comment:
Did you test this? The documentation seems to suggest that `get_event_loop` creates a new one if it doesn't exist, see: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop

Reply:
Yep, the point is that the documentation says that it generates a new event loop if doesn't exist yet and it is not being generated at that moment, but Python is also applying an extra condition (check it here https://github.com/python/cpython/blob/3.8/Lib/asyncio/events.py#L628): the thread being executed must be an instance of threading._MainThread class, but the problem is that Django (and other WSGI frameworks) are using the base class threading.Thread class (https://github.com/python/cpython/blob/3.8/Lib/threading.py#L750, which is the parent class of _MainThread) so Python is not generating a new event loop as this condition is not met.

I couldn't find any kind of information of why web frameworks are using threading.Thread class, but I realized that by creating a dummy Django app, setting a break point inside the code and checking which kind of thread is being executed:
![image](https://user-images.githubusercontent.com/12209131/83171947-1d881a00-a117-11ea-8d66-aecdee398194.png)

Reply:
UPDATE: It looks like `get_event_loop` only creates a new event loop if it's being called from the main thread. A WSGI applications may spawn a few threads for running request handlers causing `get_event_loop` to raise the exception because it's not being called from the main thread. 

So it looks like this fix is OK.